src/pages/ListPage/Filter/Filter.js

```diff
@@ -0,0 +1,298 @@
+        </div>
+      </Wrapper>
+      <Wrapper>
+        <div>
+          <Button
+            width="200px"
+            border="1px solid #e4e4e4"
+            height="38px"
+            left="0px"
+            right="20px"
+            onClick={openHeadCount}
+            position="relative"
+          >
+            인원
+          </Button>
+          {isHeadCount && (
+            <Modal position="absolute" size="20px" height="120px" width="265px">
+              <PriceRange>인원 수</PriceRange>
+              <Price left="3px">최소인원</Price>
+              <Price left="100px">최대인원</Price>
+              <PriceWrapper>
+                <PriceInput type="text" />
+                <Span margin="5px" right="5px">
+                  _
+                </Span>
+                <PriceInput type="text" />
+              </PriceWrapper>
+            </Modal>
+          )}
+        </div>
+        <div>
+          <Button
+            width="200px"
+            border="1px solid #e4e4e4"
+            height="38px"
+            left="0px"
+            right="20px"
+            onClick={openPriceRange}
+            position="relative"
+          >
+            가격 범위
+          </Button>
+          {isPriceRange && (
+            <Modal position="absolute" size="20px" height="120px" width="265px">
+              <PriceRange>가격 범위</PriceRange>
+              <Price left="3px">최저요금</Price>
+              <Price left="100px">최고요금</Price>
+              <PriceWrapper>
+                <PriceInput type="text" />
+                <Span margin="5px" right="5px">
+                  _
+                </Span>
+                <PriceInput type="text" />
+              </PriceWrapper>
+            </Modal>
+          )}
+        </div>
+        <div>
+          <Button
+            width="200px"
+            border="1px solid #e4e4e4"
+            height="38px"
+            left="0px"
+            right="20px"
+            onClick={openTheme}
+            position="relative"
+          >
+            테마
+          </Button>
+          {isTheme && (
+            <Modal
+              position="absolute"
+              size="20px"
+              display="flex"
+              wrap="wrap"
+              content="space-between"
+              width="200px"
+            >
+              <ThemeTitle> 테마 </ThemeTitle>
+              <Search margin="20px 0 10px 30px">적용하기</Search>
+              {spaceList.map(el => (
+                <Theme key={el.id} el={el} />
+              ))}
+            </Modal>
+          )}
+        </div>
+      </Wrapper>
+      <Center>
+        <Search>SEARCH </Search>
+      </Center>
+    </div>
+  );
+}
+
+const Input = styled.input`
+  border: 1px solid #e4e4e4;
+  width: 200px;
+  height: 36px;
+  border-radius: 5px;
+  font-size: 14px;
+  font-weight: 500;
+  padding: 0 10px;
+  margin-left: 10px;
+`;
+
+const Wrapper = styled.div`
+  display: flex;
+  align-items: center;
+  height: 60px;
+  border-top: ${props => props.top};
```

Comment:
```suggestion
  border-top: ${{top} => top};
```
함수 컴포넌트 사용할때와 마찬가지로 매개변수자리에서 구조분해 할당해서 사용할수도 있습니다! 참고하시고 적용해보세요!

Reply:
<img width="1440" alt="스크린샷 2021-12-15 오후 5 13 39" src="https://user-images.githubusercontent.com/73282508/146148745-e6a33d3c-b396-4921-a207-5675492ab260.png">

연욱님 이런식으로 수정했을 때 계속 오류가 나는데 제가 잘 못한 게 있을까요?

Reply:
suggestion에서 제가 문법 오류 낸게 있었네요!!
`${({top}) => top};`
이런식으로 작성해주시면 됩니다!

Reply:
> suggestion에서 제가 문법 오류 낸게 있었네요!! `${({top}) => top};` 이런식으로 작성해주시면 됩니다!

작성했습니다.